autogpt_platform/backend/.env.default

```
@@ -227,9 +227,14 @@ PLATFORM_LINK_BASE_URL=http://localhost:3000/link
 # CoPilot chat-platform bridge (Discord/Telegram/Slack)
 # Uses FRONTEND_BASE_URL (above) for link confirmation pages.
 AUTOPILOT_BOT_DISCORD_TOKEN=
-# Slack adapter — set BOTH to mount the Slack webhook routes on the main API.
+# Slack adapter — set the signing secret + client id/secret to mount the Slack
+# webhook routes and the multi-workspace "Add to Slack" install flow. The static
+# token is an optional fallback for the app's own (dev) workspace; installs from
+# other workspaces obtain their own per-team token via OAuth.
 AUTOPILOT_BOT_SLACK_TOKEN=
 AUTOPILOT_BOT_SLACK_SIGNING_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTOPILOT_BOT_SLACK_SIGNING_SECRET key should go before the AUTOPILOT_BOT_SLACK_TOKEN key

(UnorderedKey)